Reporting_SSIS/Reporting_SSIS/AdventureWorks2016CTP3.conmgr

```diff
@@ -3,6 +3,10 @@
   DTS:ObjectName="AdventureWorks2016CTP3"
```

Comment:
@Sreenitha-Sampath This should be SourceDb instead of AdventureWorks2016CTP3 as we are using params now.

Reply:
I'm done with this change